isaric/parsers/isaric-core.toml

```diff
@@ -1981,3 +2002,10 @@
   phase = "study"
   date = { field = "daily_dsstdat" }
   value = { field = "daily_fio2_lborres", apply = { function = "Percentage" } }
+
+[[observation]]
+  name = "oxygen_flow_rate"
+  phase = "study"
+  date = { field = "daily_dsstdat" }
+  value = { field = "daily_fio2c_lborres" }
+  context = ["Maximum value over 24hrs"]
```

Comment:
oxygen_cmdose can be added to oxygen_flow_rate. This is the maximum flow volume.